gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat May 20 14:19:53 MST 2017
+#Thu Jun 01 10:45:08 MST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.0-rc-1-bin.zip
```

Comment:
Concerned about tradeoffs in depending upon RC-level rather than GA-release-level dependencies. Ought uPortal to be waiting for gradle 4.0 to go GA?